vllm/model_executor/models/paligemma.py

```diff
@@ -308,9 +302,23 @@ def load_weights(self, weights: Iterable[Tuple[str, torch.Tensor]]):
             use_default_weight_loading = False
             if "vision" in name:
                 if self.vision_tower is not None:
-                    # We only do sharding for language model and
-                    # not vision model for now.
-                    use_default_weight_loading = True
```

Comment:
Please revert this for now - if we're going to apply TP on the vision tower, we should do it in another separate PR with `CLIPVisionModel` together. Ideally, we should not apply a infrastructure change to only one model.

Reply:
Reverted via dee55d0